Dockerfile

```diff
@@ -22,48 +21,30 @@ RUN apt-get update \
     libxml2-dev \
     libxslt-dev \
     zlib1g-dev \
-    python-software-properties \
     ghostscript \
     python-celery \
     python-sphinx \
-    openjdk-9-jre-headless \
-    python-virtualenv \
+    openjdk-8-jre \
     locales \
     pkg-config \
     gcc \
     libtool \
     automake
 
-RUN  locale-gen en_US.UTF-8
+RUN locale-gen en_US.UTF-8
+ENV LANG en_US.UTF-8
 
 RUN useradd -m onadata
-RUN mkdir -p /srv/onadata && chown -R onadata:onadata /srv/onadata
+RUN mkdir -p /srv/onadata && chown -R onadata:onadata /srv
 USER onadata
-RUN mkdir -p /srv/onadata/requirements
-
-ADD requirements /srv/onadata/requirements/
 
+COPY . /srv/onadata
 WORKDIR /srv/onadata
 
-RUN virtualenv /srv/onadata/.virtualenv
-RUN . /srv/onadata/.virtualenv/bin/activate; \
-    pip install pip --upgrade && pip install -r requirements/base.pip
-
-ADD . /srv/onadata/
+RUN python3.6 -m venv /srv/.virtualenv \
+  && /srv/.virtualenv/bin/pip install pip --upgrade  \
+  && /srv/.virtualenv/bin/pip install -r requirements/base.pip
 
 ENV DJANGO_SETTINGS_MODULE onadata.settings.docker
 
-USER root
-
-# for local development tmux is a nice to have
-RUN apt-get install -y tmux
-
-RUN rm -rf /var/lib/apt/lists/* \
```

Comment:
Reducing the size of the container is usually done by commands like this, probably it should be left behind.